article/performance-comparisons.tex

```diff
@@ -112,3 +112,41 @@ \subsubsection{Hyperparameter experiment}
     \caption{Recall$@$100k on NGCF PAS with different input values}
     \label{fig:recall-ngcfpas-weights}
 \end{figure}
+
+\subsubsection{Experiment with one convolution}
+An experiment was conducted to see what influence it would have on the performance of the methods to change the number of convolutions to one.
+The results from this experiment can be seen in \autoref{tab:one-convolution}.
+When comparing the results with the previous experiment in \autoref{tab:results-with-many-methods} that used 3 convolutions for all methods except PUP, it can be seen that NGCF and LightGCN have a worse performance when evaluating them with Recall@50 and NDCG@50.
```

Comment:
```suggestion
When comparing the results with the previous experiment in \autoref{tab:results-with-many-methods} that used 3 convolutions for all methods, except PUP which used 1, it can be seen that NGCF and LightGCN performs worse when evaluating them with Recall@50 and NDCG@50.
```